glTF-Toolkit.Test/Resources/gltf/BoxAnimated/BoxAnimated.gltf

```diff
@@ -0,0 +1,333 @@
+{
```

Comment:
Where is this asset from?

Reply:
https://github.com/KhronosGroup/glTF-Sample-Models